### ISSUE TITLE
Chrome dialog bug

### ISSUE THREAD

**Issue by Martinligabue** [openstreetmap/openstreetmap-website#1920]
Hi, I've been asked to send there a bug report, it's the first time I use github for this.
The bug occurs when I select "Edit with Remote Control", under Edit button in the main page, the dialog disappear instantly (if a remote client isn't setted up). It seems it only happens in Chrome.
This is an exctract from the email I recieved from Data Working Group after I sent an email to them: "At first glance it looks like a bug in your web browser.  It works fine for me (I get a modal dialogue) in Firefox and Seamonkey, but I see the same effect as you in Chrome. I guess it may be possible to work around the issue in the website code"

**Comment by tomhughes** [comment]
It's just a javascript alert so if it disappears without waiting for you to press the button then it sounds like a bug in your web browser.

**Comment by tomhughes** [comment]
In fact I never see the dialog at all in chrome because is firing the load event for the frame even though the load has failed.

Are you sure that you see the dialog and then it vanishes?

**Comment by Martinligabue** [comment]
Yes, I also have a screenshot if it helps, but it's just the dialog disappearing, nothing more

**Comment by tomhughes** [comment]
Not really, no. The whole remote editor thing is basically a pile of hacks that barely works at the best of times and if it manages to error cleanly then that's a bonus...

**Comment by Martinligabue** [comment]
maybe should it be hidden from the main "Edit" menu? Like that can be acivated from other locations, but putting something like that in the first button an user can press.. idk.
What happens now?

**Comment by mmd-osm** [comment]
Testing in current Chrome, no popup shown when JOSM is not running. Keeping the issue open.
